### PR TITLE
Issue #64 Font family changes and Bg Color changes

### LINKED ISSUE #64: Bg Color and Font family changes (Review Comments)
https://www.figma.com/file/wjOA2PNgpfOIxryT7BFlZC/Design-critique---Untitled?node-id=1%3A6&t=Au57bag1yuHuglpD-0

### PR DESCRIPTION
**Pull request by jindaliiita**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #64

Test URLs:
- Before: https://main--nedbank--hlxsites.hlx.page/
- After: https://font-bg-color-changes--nedbank--hlxsites.hlx.page/

@sdmcraft Incorporated some of. the UI Review changes. Kindly have a look.
Will create different PR's for other changes

### CODE REVIEW

File: blocks/hero-home/hero-home.less [hunk 1]
```diff
@@ -34,6 +34,11 @@ main {
 					padding: 24px 18px 40px 18px;
 					height: auto;
 					box-sizing: border-box;
+					
+					>strong {
+						font-family: @font-family-primary;
+						font-weight: @font-weight-bold;
+					}
```
Comment (sdmcraft) [inline blocks/hero-home/hero-home.less:41]: Is this style only applicable to hero block or universally applicable across site. If yes, I think we should have it is `styles.less`

File: blocks/hero-home/hero-home.less [hunk 2]
```diff
@@ -52,6 +57,11 @@ main {
 						top: 0;
 						left: 0;
 						padding: 64px 0 0 22px;
+
+						>strong {
+							font-family: @font-family-primary;
+							font-weight: @font-weight-bold;
+						}
```
Comment (sdmcraft) [inline blocks/hero-home/hero-home.less:64]: Same as previous comment.

File: blocks/hero-home/hero-home.less [hunk 3]
```diff
@@ -72,6 +82,11 @@ main {
 						padding: 98px 0 0 105px;
 						font-size: 64px;
 						line-height: 72px;
+
+						>strong {
+							font-family: @font-family-primary;
+							font-weight: @font-weight-bold;
+						}
```
Comment (sdmcraft) [inline blocks/hero-home/hero-home.less:89]: Same as previous comment.

File: blocks/columns/columns.less [hunk 4]
```diff
@@ -183,8 +185,10 @@ autocomment();
 
           >p {
             margin: 0;
-            font-size: 20px;
+            font-size: 18px;
```
Comment (sdmcraft) [inline blocks/columns/columns.less:188]: We can take this as an opportunity to use the design token here from `variables.less`

**Review by sdmcraft — COMMENTED** [review]
Thanks. General comment,

**Comment by jindaliiita** [comment]
@sdmcraft Can you please approve this

File: styles/styles.less [hunk 5]
```diff
@@ -106,15 +106,27 @@ h3,
...
+h3 {
+	font-weight: @font-weight-bold;
+	font-family: @font-family-primary;
+}
+
+h4,
+h5,
+h6 {
+	font-weight: 500;
+	font-family: var(--heading-font-family);
```
Comment (sdmcraft) [inline styles/styles.less:127]: Do we need different font for `h4, h5, h6`  from what's there for `h1, h2 , h3`?

Comment (jindaliiita) [reply styles/styles.less:127]: Yes i have checked their website and also they have asked to changed only for these

File: styles/styles.less [hunk 6]
```diff
@@ -168,7 +176,8 @@ pre {
 }
 
 strong {
-	font-family: var(--strong-font-family);
+	font-family: @font-family-primary;
```
Comment (sdmcraft) [inline styles/styles.less:179]: I think we should not explicitly specify this and let it inherit from it parent.

**Review by sdmcraft — APPROVED** [review]

**Pull request merged**